dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -148,6 +148,7 @@
 -Des.discovery.zen.ping.multicast.enabled=false
 -Des.script.inline=on
 -Des.script.indexed=on
+-Des.node.enable_custom_paths=true
```

Comment:
We could probably remove `enable_custom_paths` now, but maybe as a separate change?